cli/actions/export_test_action.go

```diff
@@ -69,13 +69,14 @@ func (a exportTestAction) getDefinitionFromServer(ctx context.Context, args Expo
 		expectedVersion = *test.Version
 	}
 
-	request := a.client.ApiApi.GetTestVersionDefinitionFile(ctx, args.TestId, expectedVersion)
-	definitionString, _, err := a.client.ApiApi.GetTestVersionDefinitionFileExecute(request)
-	if err != nil {
-		return "", fmt.Errorf("could not get test definition: %w", err)
-	}
+	// request := a.client.ApiApi.GetTestVersionDefinitionFile(ctx, args.TestId, expectedVersion)
+	// definitionString, _, err := a.client.ApiApi.GetTestVersionDefinitionFileExecute(request)
+	// if err != nil {
+	// 	return "", fmt.Errorf("could not get test definition: %w", err)
+	// }
 
-	return definitionString, nil
+	// return definitionString, nil
+	return "", fmt.Errorf("deprecated")
```

Comment:
Are we sending any message to the user saying to use `get` instead of `export` now?

Reply:
no, this will be fixed in a follow up PR